cmd/git-sync/main.go

```diff
@@ -262,6 +305,16 @@ func updateSymlink(gitRoot, link, newDir string) error {
 		log.V(1).Infof("pruned old worktrees")
 	}
 
+	// Calling webhook only after new symlink created - one after another
+	for _, v := range *WebhookArray {
```

Comment:
for discussion - should this be async?  If the webhook is slow, the main loop stops.  Maybe a single goroutine and a channel of struct{}